Tests/LibWeb/Text/input/css/filter-grayscale.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<style>
+    #high{
+        background-color: blue;
+        height: 30px;
+        width: 30px;
+        filter: grayscale(200%) brightness(200%) contrast(200%) invert(200%) opacity(200%) saturate(200%) sepia(200%);
+    }
+    #default{
+        background-color: blue;
+        height: 30px;
+        width: 30px;
+        filter: grayscale() brightness() contrast() invert() opacity() saturate() sepia();
```

Comment:
Why not include functions like `hue-rotate` and `drop-shadow` as well?

Reply:
Because I only touched the filters implemented as "color" filters, which is exactly this subset.

Reply:
Fair enough! That's a FIXME then.